src/components/RegisterForm/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Input, Button, Select } from 'antd';
+import { NavLink } from 'react-router-dom';
+
+import './style.less';
+import logo from '../../assets/graphics/logo.svg';
+
+const { Option } = Select;
+
+const RegisterForm = props => {
+  return (
+    <div className="registercard">
+      <div className="content">
+        <img src={logo} alt="logo" height="115" width="115" />
+        <h1>Register for ACM@UCSD</h1>
+        <form onSubmit={props.handleSubmit}>
+          <div className="names">
+            <Form.Item label="First Name" className="formitems">
+              <Input
+                name="firstname"
+                className="firstname"
+                value={props.values}
+                onChange={props.handleChange}
+                onBlur={props.handleBlur}
+              />
+            </Form.Item>
+            <Form.Item label="Last Name" className="formitems">
+              <Input
+                name="lastname"
+                className="lastname"
+                value={props.values}
+                onChange={props.handleChange}
+                onBlur={props.handleBlur}
+              />
+            </Form.Item>
+          </div>
+          <Form.Item label="Email (user@ucsd.edu)" className="formitems">
+            <Input
+              name="email"
+              type="email"
+              className="input-box"
+              value={props.values}
+              onChange={props.handleChange}
+              onBlur={props.handleBlur}
+            />
+          </Form.Item>
+          <Form.Item label="Password" className="formitems">
+            <Input
+              name="password"
+              type="password"
+              className="input-box"
+              value={props.values}
+              onChange={props.handleChange}
+              onBlur={props.handleBlur}
+            />
+          </Form.Item>
+          <Form.Item label="Confirm password" className="formitems">
+            <Input
+              name="confirmpassword"
+              type="password"
+              className="input-box"
+              value={props.values}
+              onChange={props.handleChange}
+              onBlur={props.handleBlur}
+            />
+          </Form.Item>
+          <div className="majoryear">
+            <Form.Item label="Major" className="formitems">
+              <Input
+                name="major"
+                className="major"
+                value={props.values}
+                onChange={props.handleChange}
+                onBlur={props.handleBlur}
+              />
+            </Form.Item>
+            <Form.Item label="Year" className="formitems">
+              <Select className="year" value={props.values}>
+                <Option value="2020">2020</Option>
```

Comment:
can you make this an array at the top and then utilize a for loop to iterate through it? By doing it that way, it'll be convenient to add more values when we have to later on.